centreon/www/class/centreon-clapi/centreonCentbrokerCfg.class.php

```
@@ -183,432 +233,278 @@ public function show($parameters = null, $filters = []): void
+                        = $this->action . $this->delim . 'ADD' . strtoupper($row['config_group'])
+                        . $this->delim . $element['config_name']
+                        . $this->delim . $row['config_value'];
+                } elseif ($row['config_key'] == 'blockId') {
+                    $blockId[$row['config_group'] . '_' . $row['config_group_id']] = $row['config_value'];
+                } elseif ($row['config_key'] == 'category') {
+                    $categories[$row['config_group'] . '_' . $row['config_group_id']][] = $row['config_value'];
+                }
+            }
+            foreach ($addParamStr as $id => $add) {
+                if (isset($blockId[$id], $setParamStr[$id])) {
+                    [$tag, $type] = explode('_', $blockId[$id]);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$tag'. (Unused Code Rules)

(UnusedLocalVariable)